scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
```

Comment:
:stop_sign: **Security Vulnerability**: Command injection vulnerability detected. Directly using user input in `child_process.exec()` can lead to arbitrary command execution[^1]. Always sanitize and validate user input before using it in system commands.
```suggestion
// Command injection vulnerability mitigation
function executeCommand(userInput: string) {
    const sanitizedInput = userInput.replace(/[;&|`$()]/g, '');
    child_process.execFile('ls', [sanitizedInput], (error, stdout, stderr) => {
        if (error) {
            console.error(`exec error: ${error}`);
            return;
        }
        console.log(`stdout: ${stdout}`);
        console.error(`stderr: ${stderr}`);
    });
}
```

[^1]: CWE-78: Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection') - https://cwe.mitre.org/data/definitions/78.html